src/controllers/expenses.js

```diff
@@ -0,0 +1,119 @@
+'use strict';
+
+const {
+  getAllExpenses,
+  addExpense,
+  getExpense,
+  deleteExpense,
+  updateExpense,
+} = require('../services/exprensesService');
+const { getAllUsers } = require('../services/usersServer.js');
```

Comment:
There is a typo in the required file 'usersServer.js', it should be 'usersService.js'

Reply:
done